tests/collections/asr/test_asr_rnnt_encdec_model.py

```diff
@@ -73,7 +73,7 @@ def predict(
                 return (
                     output,
                     [
-                        torch.tensor([0] * self.vocab_size + [1], dtype=torch.float32)[None, None, :].exand(
+                        torch.tensor([0] * self.vocab_size + [1], dtype=torch.float32)[None, None, :].expand(
```

Comment:
How how did this test pass with this error ?

Reply:
I think this case is redundant and never executed in decoding, but we need to implement this to conform the interface where `y` is optional (see `AbstractRNNTDecoder.predict`)